chronos/__init__.py

```diff
@@ -63,6 +70,12 @@ def __init__(self, servers, proto="http", username=None, password=None, extra_he
             self._password = password
         logging.basicConfig(format='%(asctime)s %(levelname)-8s %(message)s', level=level)
         self.logger = logging.getLogger(__name__)
+        if scheduler_version is None:
+            raise ChronosAPIError("Chronos >=3.x requires scheduler_version set")
+        else:
+            if scheduler_version not in SCHEDULER_VERSIONS:
+                raise ChronosAPIError('Wrong scheduler_version provided')
+            self._prefix = "/%s" % (scheduler_version,)
```

Comment:
I think you need to add empty-string ``''`` to list list of acceptable ``SCHEDULER_VERSIONS``, or allow None so people can keep using < 3